src/main/java/net/io/kino/model/Showtime.java

```diff
@@ -17,10 +17,10 @@
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
 
-    @OneToOne
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
Z tymi kaskadami to bym nie przesadzał. W tym momencie to oznacza, że jak usunie się `Showtime` to automatycznie `Movie` jest też usuwany, co nie zawsze jest pożądaną sytuacją.